python2/httplib2/__init__.py

```diff
@@ -373,6 +373,15 @@ def _normalize_headers(headers):
     )
 
 
+def _formalize_headers(headers):
```

Comment:
Why is this needed?

Reply:
It's to compatible with an unstandardized web server in my device,  introduced to this PR by mistake, I have removed it.